### DISCUSSION TITLE
Nuclei can't update templates

### DISCUSSION THREAD

**Discussion by littleheary** [projectdiscovery/nuclei#1460] [General]
i use nuclei 2.5.4. and 2.5.7,when i update template,can't success.

$ nuclei.exe -ut

                     __     _
   ____  __  _______/ /__  (_)
  / __ \/ / / / ___/ / _ \/ /
 / / / / /_/ / /__/ /  __/ /
/_/ /_/\__,_/\___/_/\___/_/   2.5.7

                projectdiscovery.io

[WRN] Use with caution. You are responsible for your actions.
[WRN] Developers assume no liability and are not responsible for any misuse or damage.
[INF] No new updates found for nuclei templates

but my local templates is 2021.12.10,latest version is 2021.1

please check this reason.

**Comment by ehsandeep** [comment]
The latest nuclei template version is [v8.8.1](https://github.com/projectdiscovery/nuclei-templates/releases/tag/v8.8.1), what do you mean by `2021.12.10` or `2021.1`?
